diff --git a/node_modules/@capacitor-community/media/android/src/main/java/com/getcapacitor/community/media/MediaPlugin.java b/node_modules/@capacitor-community/media/android/src/main/java/com/getcapacitor/community/media/MediaPlugin.java
index ff205a4..b93b528 100644
--- a/node_modules/@capacitor-community/media/android/src/main/java/com/getcapacitor/community/media/MediaPlugin.java
+++ b/node_modules/@capacitor-community/media/android/src/main/java/com/getcapacitor/community/media/MediaPlugin.java
@@ -381,7 +381,7 @@ public class MediaPlugin extends Plugin {
                 call.reject("Cant create album", EC_FS_ERROR);
             } else {
                 Log.d("DEBUG LOG", "___SUCCESS ALBUM CREATED");
-                call.success();
+                call.resolve();
             }
         } else {
             Log.d("DEBUG LOG", "___ERROR ALBUM ALREADY EXISTS");
